app/views/pages/index.html.erb

```diff
@@ -1,4 +1 @@
-<h1>Pages#index</h1>
-<p>Find me in app/views/pages/index.html.erb</p>
-<!-- erb: paste this in view -->
 <%= react_component("App") %>
```

Comment:
- [ ] Your application's API is working as expected. But your frontend is not showing the react component. Can you please take a look at it.

Please see test demo of your app to see what I am try to point out here.

Demo|
:-------:|
![a1](https://user-images.githubusercontent.com/39523312/164520237-8fad3621-af05-402d-966d-7874f9bf42d2.gif)

Reply:
Thank you for the feedback, setup branch for pull request does not have commented code.  So may be because of running in that branch, it is showing blanks. In my side, app is running fine. Thank you.
[app/views/pages/index.html.erb](https://github.com/aakbarkhan/Rails-helloworld/pull/1/files/b371094fbfe1a7ae1f089489612068f570f344d5#diff-61970285059c3fc9b5fe2d5b51fe729898101c7a0b3bbefff60f909badf10ed5)